### PR TITLE
GH-4367 Clarify Readme for deps & commandline build

### LINKED ISSUE #4367: Build Errors When Cloning Code and Running MSBuild
### Description
I was following the directions on the README and running the msbuild command gave the following errors:

https://gist.github.com/jamesmontemagno/a14a259c965a6e37301a0231d2fd1365

### Steps to Reproduce

1. Fork code
2.  Clone Locally
3. run `msbuild /restore Xamarin.Forms.sln`

### PR DESCRIPTION
**Pull request by jamesmontemagno**
### Description of Change ###

Ensure everyone knows to install .NET 4.7 dependency and also simplify msbuild into single action!

### Issues Resolved ### 

- fixes #4367

### API Changes ###
None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,9 +45,12 @@ If you want to use the latest dev build then you should read [this blog post]( h
 ##### Install Visual Studio 2017 #####
 VS 2017 is required for developing Xamarin.Forms. If you do not already have it installed, you can download it [here](https://www.visualstudio.com/downloads/download-visual-studio-vs). VS 2017 Community is completely free. If you are installing VS 2017 for the first time, select the "Custom" installation type and select the following from the features list to install:
 
+- .NET desktop development - In the `Summary > Optional select .NET Framework 4.7 SDK, .NET Framework 4.7 targeting pack`. 
 - Universal Windows Platform Development - In the `Summary > Optional select the Windows 10 Mobile Emulator`.
 - Mobile Development with .NET - In the `Summary > Optional select Xamarin Remoted Simulator, Xamarin SDK Manager, Intel Hardware Accelerated Execution Manager (HAXM)`
 
+The Android 7.1 Nougat API 24 SDK is required for developing Xamarin.Forms. It can be installed by using the [Xamarin Android SDK Manager](https://docs.microsoft.com/xamarin/android/get-started/installation/android-sdk).
```
Comment (rmarinho) [inline README.md]: maybe 27 ?

**Comment by jamesmontemagno** [comment]
@rmarinho is it required to run the `nuget restore` command? I added it into `msbuild` because when I tried to run `nuget` it said no.

File: README.md (hunk 1 above)
Comment (jamesmontemagno) [reply README.md]: Great question! I know that you have to have SDK 24 installed because it yelled at me! perhaps also SDK 27? we should confirm.

**Comment by hartez** [comment]
@jamesmontemagno Can you resolve the conflicts? Thanks!

**Comment by jamesmontemagno** [comment]
Done :)

**Pull request merged**